### ISSUE TITLE
pretrained S/4 Chair model's performance

### ISSUE THREAD

**Issue by LiyuRen00** [DiT-3D/DiT-3D#16]
May I ask why the result of using V100 16G to validate the pre trained model is 1-NN-CD: 64.50 1-NN-EMD: 63.75 COV-CD: 39.88 COV-EMD: 43.05 instead of the expected 1-NNA-CD: 56.31 1-NNA-EMD: 55.82 COV-CD: 47.21 COV-EMD: 50.75 in the ReadMe

**Comment by DiT-3D** [comment]
Hi, thanks for your interest in our work! Please follow README step by step to achieve the expected results.

**Comment by JudgeLJX** [comment]
> May I ask why the result of using V100 16G to validate the pre trained model is 1-NN-CD: 64.50 1-NN-EMD: 63.75 COV-CD: 39.88 COV-EMD: 43.05 instead of the expected 1-NNA-CD: 56.31 1-NNA-EMD: 55.82 COV-CD: 47.21 COV-EMD: 50.75 in the ReadMe

Hi  Did you achieved their results?

**Comment by kehuantiantang** [comment]
I have the same problem, I use S/4 model to train 'chair', the loss decrease very quickly in the begining of 100 epochs. Then, eventhough I change learning rate or optimizer, the loss cannot decrease any more ....... Is there any suggesstion ? Thanks  a lot ?

<img width="1334" height="576" alt="Image" src="https://github.com/user-attachments/assets/9917d151-06b9-45eb-8f38-e4b2f60c2134" />

**Comment by LukeBriton** [comment]
The pre-trained model tested result on my server is:
```
2025-12-30 17:22:36,584 : {'1-NNA-CD': np.float64(0.5357050624760714), '1-NNA-EMD': np.float64(0.5205965922637419), 'COV-CD': np.float64(0.5095557841387662), 'COV-EMD': np.float64(0.5520402897487987), 'MMD-CD': np.float64(0.004831254778599197), 'MMD-EMD': np.float64(0.02627357425676151), 'JSD': np.float64(0.15865010989604836)}
```
